rviz_rendering/src/rviz_rendering/ogre_render_window_impl.cpp

```diff
@@ -231,14 +231,14 @@ RenderWindowImpl::initialize()
   }
 
   if (ogre_camera_) {
+    ogre_camera_->setAspectRatio(
+      Ogre::Real(parent_->width()) / Ogre::Real(parent_->height()));
+    ogre_camera_->setAutoAspectRatio(true);
+
```

Comment:
Is there a behavior change by move these calls earlier or just an aesthetic change?

Reply:
there should be no behavior change in rviz by this pull request. I just was doing a deep dive in the code and figured I would try and clean it up a bit to make it easier to understand

Reply:
I guess it didn't really though based on the questions on this PR...